siriuspy/siriuspy/sofb/utils.py

```diff
@@ -65,6 +71,7 @@ def si_calculate_bump(orbx, orby, subsec, agx=0, agy=0, psx=0, psy=0):
     vec = _np.array([psx, agx, psy, agy])
     pos_bpm = _np.dot(BUMP_MATRICES[subname], vec)
 
+    # NOTE: does this not work for subsec '01SA' ?
```

Comment:
It does work for 01SA. I think we could remove this comment.

Reply:
of course! index -1 lead to last element!